{"links": {"self": {"href": "data/repositories/hoffmanlab/segway/issues/32/comments/18597697.json"}, "html": {"href": "#!/hoffmanlab/segway/issues/32#comment-18597697"}}, "issue": {"links": {"self": {"href": "data/repositories/hoffmanlab/segway/issues/32.json"}}, "type": "issue", "id": 32, "repository": {"links": {"self": {"href": "data/repositories/hoffmanlab/segway.json"}, "html": {"href": "#!/hoffmanlab/segway"}, "avatar": {"href": "data/bytebucket.org/ravatar/{68bf38bf-f0c1-4c9c-909e-0ee4b2c26257}ts=python"}}, "type": "repository", "name": "segway", "full_name": "hoffmanlab/segway", "uuid": "{68bf38bf-f0c1-4c9c-909e-0ee4b2c26257}"}, "title": "Error: `traindir/likelihood/likelihood.0.ll` and `traindir/likelihood/likelihood.0.ll` are the same file"}, "content": {"raw": "You are correct. It seems clear that the destination likelihood file is always 'traindir/likelihood/likelihood.0.ll' which is the default since `save_observation_params` performs the following:\n```python\n        if self.train:\n            self.set_log_likelihood_filenames()\n```\nWhich produces the default '0' numbered destination likelihood filename. Do you know why the `copy2(src_filename, dst_filename)` doesn't fail in the single threaded case? Surely they would both have their likelihood filenames numbered at '0'. Why doesn't this crash?\n", "markup": "markdown", "html": "<p>You are correct. It seems clear that the destination likelihood file is always 'traindir/likelihood/likelihood.0.ll' which is the default since <code>save_observation_params</code> performs the following:</p>\n<div class=\"codehilite language-python\"><pre><span></span>        <span class=\"k\">if</span> <span class=\"bp\">self</span><span class=\"o\">.</span><span class=\"n\">train</span><span class=\"p\">:</span>\n            <span class=\"bp\">self</span><span class=\"o\">.</span><span class=\"n\">set_log_likelihood_filenames</span><span class=\"p\">()</span>\n</pre></div>\n\n\n<p>Which produces the default '0' numbered destination likelihood filename. Do you know why the <code>copy2(src_filename, dst_filename)</code> doesn't fail in the single threaded case? Surely they would both have their likelihood filenames numbered at '0'. Why doesn't this crash?</p>", "type": "rendered"}, "created_on": "2015-06-04T18:34:10.874529+00:00", "user": {"display_name": "Eric Roberts", "uuid": "{cd8c1fe0-28ca-45fb-8ef9-48090b42bb80}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bcd8c1fe0-28ca-45fb-8ef9-48090b42bb80%7D"}, "html": {"href": "https://bitbucket.org/%7Bcd8c1fe0-28ca-45fb-8ef9-48090b42bb80%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/17548d71dfb29d015b880f48cfc01ca3d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsER-5.png"}}, "nickname": "ericr86", "type": "user", "account_id": "557058:c80ca578-03a1-4ac6-b3ee-50372a3fceee"}, "updated_on": null, "type": "issue_comment", "id": 18597697}